cmd/ndm_daemonset/app/command/commands.go

```diff
@@ -49,22 +39,17 @@ func NewNodeDiskManager() (*cobra.Command, error) {
 		},
 	}
 
+	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
+
 	// add the glog flags
-	cmd.PersistentFlags().AddGoFlagSet(flag.CommandLine)
-	//add flag for /metrics endpoint port and endpoint path
-	cmd.PersistentFlags().String("port", ":9090", "Port to launch HTTP server.")
-	cmd.PersistentFlags().String("metricspath", "/metrics", "Endpointpath to get metrics.")
+	cmd.PersistentFlags().AddGoFlagSet(goflag.CommandLine)
```

Comment:
This line is not required, as we have already added the flags to `pflag`.

Reply:
Handled as part of https://github.com/openebs/node-disk-manager/pull/325/commits/dc4751cb1ad1e0ae3f3c44080879ca6cc57eeab3